### PR TITLE
Adding Gibon pool

### PR DESCRIPTION
**Pull request by gibonada**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+      "poolId":"b352315bd54d61dfcd8fd78a2aa4f3e0cc8ef342b9495465b0b0b2ce",
+      "social":{
+         "facebook":"",
+         "twitter":"@gibon_pool",
+         "website":"https://gibon.online",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"https://t.me/gibonpool",
```
Comment (leo42) [inline registry.json:13]: We need a user telegram handle, no groups of channels please. this is for voting purpose's.

**Review by leo42 — CHANGES_REQUESTED** [review]

**Review by gibonada — COMMENTED** [review]
Updating my telegram handle

**Comment by gibonada** [comment]
Hi, I hope I update it, my telegram handle is @GibonAda, thanks.
Have a nice day Paul.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
neděle 30. květen 2021 17:02, leo42 ***@***.***> napsal:

**Comment by gibonada** [comment]
Hi Leo,
could you please add my pool to registry? Its no longer on the list.
{

"ticker":"GIBON",

"poolId":"b352315bd54d61dfcd8fd78a2aa4f3e0cc8ef342b9495465b0b0b2ce",

"social":{

"facebook":"",

***@***.***_pool",

"website":"https://gibon.online",

"reddit":"",

"instagram":"",

"youtube":""

},

"telegram":"@GibonAda",

"operatorName":"Paul",

"github":""

},
I cannot merge it, and it always conflicting. Thank you very much. Paul.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
pátek 4. červen 2021 22:47, Pavel Vahala ***@***.***> napsal:

**Pull request closed without merging**